### PR TITLE
Show and update notification

### PR DESCRIPTION
**Pull request by nvs266**

### CODE REVIEW

File: app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java [hunk 1]
```diff
@@ -19,17 +30,32 @@
     public static final String ACTION_PREVIOUS_TRACK = "ACTION_PREVIOUS_TRACK";
 
     // Notification
-    public static final int NOTIFY_ID = 1;
+    private final int NOTIFY_ID = 1;
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java]: constant khai báo private static final

File: app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java [hunk 2]
```diff
@@ -113,4 +147,106 @@ public MusicService getService() {
...
+    public void initializeNotificationPause() {
+        if (getCurrentTrack() == null) return;
+        stopForeground(false);
+
+        initializeNotificationBase();
+        mBuilder = new NotificationCompat.Builder(getApplicationContext(), "")
+                .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
+                .setSmallIcon(R.drawable.ic_music_note_deep_orange_500_24dp)
+                .setContentIntent(pendingIntentOpenApp)
+                .addAction(R.drawable.ic_skip_previous_black_24dp, "Previous", prevPendingIntent)   // #0
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java]: hardcode

File: app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java [hunk 3]
```diff
@@ -113,4 +147,106 @@ public MusicService getService() {
...
+                .addAction(R.drawable.ic_play_arrow_black_36dp, "Play", ptPlayPause)    // #1
+                .addAction(R.drawable.ic_skip_next_black_24dp, "Next", nextPendingIntent)   // #2
+                .setOngoing(false) //   Can swipe notification
+                .setStyle(new android.support.v4.media.app.NotificationCompat.MediaStyle()
+                        .setShowActionsInCompactView(0, 1, 2))  // #0 - #1 - #2
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java]: hardcode

File: app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java (hunk 1 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java]: Resolved

File: app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java (hunk 2 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java]: Resolved

File: app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java (hunk 3 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/mysoundcloud/service/MusicService.java]: Resolved

**Pull request merged**